### PR TITLE
docs: split Lint and Test badges for title asdf in `README.MD`

### PR DESCRIPTION
**Pull request by ooooo-create**
# Summary

Fixes: #1688 

## Other Information

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,20 +44,20 @@ See [CONTRIBUTING.md in the repo](https://github.com/asdf-vm/asdf/blob/master/CO
...
-> 
+>
 > Then I said, just one more version manager<br/>
 > Which I will write instead
-> 
+>
 > So, there came another version manager<br/>
 > **asdf version manager** - <https://github.com/asdf-vm/asdf>
-> 
+>
```
Comment (sparr) [inline README.md:60]: I think `> ` is the canonical indent here and the spaces should be preserved.

Comment (ooooo-create) [reply README.md:60]: Thanks! I agree with your point that `> ` is the canonical indent here and the spaces should be preserved.

**Review by sparr — APPROVED** [review]
Thanks for this. I hope this gets accepted.

**Review by Stratus3D — APPROVED** [review]
Sorry for the late review @ooooo-create! Thanks for the PR.

**Pull request merged**